fs_src/index.html

```diff
@@ -286,8 +286,8 @@ <h1 id="head">Switch</h1>
           <span class="slider round"></span>
         </label>
       </div>
-      <div>
-        <label for="auto_off_delay">Auto off delay:</label>
+      <div class="form-control" id="auto_off_delay_container">
+        <label for="auto_off">Auto off delay:</label>

```

Comment:
input element is still "auto_off", so why change this?

Reply:
sorry still learning js, is this now correct https://github.com/mongoose-os-apps/shelly-homekit/pull/490/commits/71c9bff14f63aeb3bb077c31c3ed1fc28872ea37 ?

Reply:
yes, looks good